tests/tasks/test_tasks.py

```diff
@@ -19,6 +21,20 @@
 TIME_LIMIT = 60
```

Comment:
Is there a reasoning for this particular timeout? And do we really want to set the same timeout for file downloads and task completion - which are quite different processes?

Reply:
You're right it doesn't make sense to have the same time limit for each process. I think I'm going to change the decorator to take a parameter (e.g, `@timeout(30)`). I chose 60 seconds arbitrarily (I figured the user could change it based on how fast/slow the TES instance they're using is). Do you have any suggestions on what how to pick a timeout (e.g., 5 seconds more than average run time)?

Reply:
Yes, a param is a good solution. I guess the timeout default depends on the variability of the distribution of the run time. I would put it at something like 3 standard deviations away, then you'll get a timeout less than 1% of the time (unless sth is wrong). So if the average runtime is 20 + 3 seconds, I would put it at ~30 seconds.